config/routes.rb

```diff
@@ -287,7 +287,7 @@
       end
 
       scope 'applications/:application_id', :as => :application do
-        resources :keys, constraints: { id: %r{[^/]+} }, only: %i[new create edit update destroy] do
+        resources :keys, constraints: { id: /.+/ }, only: %i[new create edit update destroy] do
```

Comment:
Wouldn't this be equivalent to

```suggestion
        resources :keys, only: %i[new create edit update destroy] do
```

Reply:
@akostadinov No, because with no constraints Rails doesn't allow dots in parameter values... 
This is even outlined in the official docs: https://guides.rubyonrails.org/routing.html#specifying-constraints